### PR TITLE
[docs] Add llvm and clang release notes for the global-var code model attribute

### PR DESCRIPTION
**Pull request by heiher**

### CODE REVIEW

File: llvm/docs/ReleaseNotes.rst [hunk 1]
```diff
@@ -70,6 +70,8 @@ Changes to the LLVM IR
 
 * Added `llvm.exp10` intrinsic.
 
+* Added a code model attribute for the global variable.
```
Comment (arsenm) [inline llvm/docs/ReleaseNotes.rst:73]: Helpful to include link to the LangRef reference

Comment (heiher) [reply llvm/docs/ReleaseNotes.rst]: Thanks

**Review by SixWeining — APPROVED** [review]
LGTM. Thanks.

File: llvm/docs/ReleaseNotes.rst [hunk 2]
```diff
@@ -130,6 +132,8 @@ Changes to the Hexagon Backend
 
 Changes to the LoongArch Backend
 --------------------------------
+* Respect the code model attribute of global variables when different from
+  their default values.
```
Comment (xen0n) [inline llvm/docs/ReleaseNotes.rst:136]: "Code model of global variables can now be overridden by means of the newly-added `code_model` LLVM IR attribute."?

File: llvm/docs/ReleaseNotes.rst [hunk 3]
```diff
@@ -70,6 +70,8 @@ Changes to the LLVM IR
 
 * Added `llvm.exp10` intrinsic.
 
+* Added a code model attribute for the `global variable <LangRef.html#global-variables>`_.
```
Comment (xen0n) [inline llvm/docs/ReleaseNotes.rst:73]: Refer to the IR construct verbatim, i.e. `code_model`, for better readability?

File: clang/docs/ReleaseNotes.rst [hunk 4]
```diff
@@ -1119,6 +1119,14 @@ Windows Support
 
 LoongArch Support
 ^^^^^^^^^^^^^^^^^
+- Added a code model attribute for the global variable. The following values
```
Comment (xen0n) [inline clang/docs/ReleaseNotes.rst:1122]: "The `model` attribute is now supported for overriding the default code model used to access global variables." or something like that?

Comment (heiher) [reply clang/docs/ReleaseNotes.rst]: Thanks.

**Pull request merged**